ssh/ssh_tunnel.py

```diff
@@ -88,13 +88,13 @@ def write_to_remote(self, src, dst):
             dst: (str) destination for path
         """
         cmd = self.ssh_cmd + ['-C', '-p', str(self.port), self.target, 'cat>' + dst]
-        logger.debug('Running socket write: ' + ' '.join(cmd))
+        log.info('Running socket write: ' + ' '.join(cmd))
         with open(src, 'r') as fh:
             check_call(cmd, stdin=fh)
 
     def close(self):
         close_tunnel = self.ssh_cmd + ['-p', str(self.port), '-O', 'exit', self.target]
-        logger.debug('Closing SSH Tunnel: ' + ' '.join(close_tunnel))
+        log.debug('Closing SSH Tunnel: ' + ' '.join(close_tunnel))
```

Comment:
I think this should become log.info() as well. We log.info() the creation, and if this happens unexpectedly early bad things happen.